### ISSUE TITLE
Console error in welcome script

### ISSUE THREAD

**Issue by TRManderson** [UQComputingSociety/uqcsbot#150]
```
error:  TypeError: Cannot read property 'members' of undefined
  at /srv/hubot/scripts/welcome.js:33:39
```

**Comment by mitchmcdee** [comment]
Yeah this was the reason milestones weren't working. Basically the users.list web API was returning a promise but the code was giving it a callback with undefined args, thus the error

This was addressed in https://github.com/UQComputingSociety/uqcsbot/pull/166, will close when that's been merged :)

**Comment by mitchmcdee** [comment]
Fixed in 421bcc4a631f2dce0e2bccb58a8a24b14558085c, closing
